lib/code_corps/stripe_service/stripe_connect_subscription_service.ex

```diff
@@ -20,19 +18,22 @@ defmodule CodeCorps.StripeService.StripeConnectSubscriptionService do
 
   - `{:ok, %StripeConnectSubscription{}}` - the created record.
   - `{:error, %Ecto.Changeset{}}` - the record was not created due to validation issues.
+  - `{:error, :project_not_found}`
   - `{:error, :project_not_ready}` - the associated project does not meed the prerequisites for receiving donations.
+  - `{:error, :user_not_found}`
   - `{:error, :user_not_ready}` - the associated user does not meet the prerequisits to donate.
   - `{:error, %Stripe.APIErrorResponse{}}` - there was a problem with the stripe request
-  - `{:error, :not_found}` - one of the associated records was not found
 
   # Side effects
 
   - If the subscription is created or found, associated project totals will get updated
   - If the subscription is created or found, associated donation goal states will be updated
   """
-  def find_or_create(%{"project_id" => project_id, "quantity" => _, "user_id" => user_id} = attributes) do
-    with {:ok, %Project{} = project} <- get_project(project_id) |> ProjectSubscribable.validate,
-         {:ok, %User{} = user} <- get_user(user_id) |> UserCanSubscribe.validate
+  def find_or_create(%{"project_id" => project_id, "user_id" => user_id, "quantity" => _} = attributes) do
```

Comment:
Would alpha-order the attrs `%{"project_id" => project_id, "quantity" => _, "user_id" => user_id}`